### ISSUE TITLE
Work fine with LeafletJS 1.7.1

### ISSUE THREAD

**Issue by piergiorgio-roveda** [ewoken/Leaflet.MovingMarker#39]
Hi, I create a map with this plugin and work fine [mappa-milano-2020/template04](https://www.cityplanner.biz/mappa-milano-2020/template04/)

**Comment by iamnotacoder-djs** [comment]
Thats true. I used LeafletJS 1.3.1, so I was getting errors trying to attach MovingMarker.js. Now works fine on 1.7.1. 
Thanks for information. [Here is my demo](https://apexmap.online/?movingmarker) 
And… nice demo C:

![mappa-milano-2020/template04](https://i.imgur.com/2SUl2Io.png)

**Comment by piergiorgio-roveda** [comment]
I'm sorry my demo didn't work. Now I've fixed it. Your demo is beautiful too!
![image](https://user-images.githubusercontent.com/74056468/127624014-76a2bb3b-ea33-41d0-bdff-3da684e39721.png)
